entity-service/src/main/resources/neo4j/migrations/V06__remove_dangling_relationships.cypher

```diff
@@ -0,0 +1,3 @@
+MATCH (rel:Relationship)
```

Comment:
I'm not totally sure about this migration script.
The fact that we match all relationships and we delete those not pointing to Entity, are we sure that all relationships in the graph (should) target entities ?  
Also it fails in local with ConstraintValidationFailed error.

Reply:
Hmm, yes, a relationship node always targets an entity (that's the NGSI-LD data model), are you thinking about something specific?

I gonna check for the constraint error.

Reply:
Damn, partial entities, I forgot about them...

Reply:
OK, updated

Reply:
Yes I was thinking about Partial Entities.
LGTM